source/package-json.d.ts

```diff
@@ -224,7 +224,10 @@ declare namespace PackageJson {
 	string
 	>;
 
-	type ExportConditions = {[condition in ExportCondition]: Exports};
+	/**
+	A mapping of conditions and the paths to which they resolve.
+	*/
+	type ExportConditions = {[condition in ExportCondition]?: Exports};
```

Comment:
The `?` here introduces `undefined` as a valid exports value in the type, which is not possible by JSON.